modules/openapi-generator/src/main/java/org/openapitools/codegen/config/GeneratorProperties.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ * Copyright 2018 SmartBear Software
```

Comment:
For new files, there's no need to include line 3. Please refer to https://opensource.stackexchange.com/questions/7300/copyright-notice-in-the-file-header-apache-v2-license/7301#7301 for more information.

Reply:
@gndrm after you remove line 3, we'll be happy to merge your contributions to the master.

Have a nice weekend.

Reply:
done, sorry for the late reply. was very short on time the last weeks.